plugins/depthCamera/src/DepthCameraDriver.cpp

```diff
@@ -120,23 +127,25 @@ bool GazeboYarpDepthCameraDriver::close()
     // Copy & paste like a Monkey
     if (this->m_updateImageFrame_Connection.get())
     {
-        m_imageCameraPtr->DisconnectNewImageFrame(this->m_updateImageFrame_Connection);
+        m_depthCameraPtr->DisconnectNewImageFrame(this->m_updateImageFrame_Connection);
         this->m_updateImageFrame_Connection = gazebo::event::ConnectionPtr();
     }
 
     if (this->m_updateRGBPointCloud_Connection.get())
     {
-        m_imageCameraPtr->DisconnectNewImageFrame(this->m_updateRGBPointCloud_Connection);
+        m_depthCameraPtr->DisconnectNewImageFrame(this->m_updateRGBPointCloud_Connection);
```

Comment:
Exactly. The interface has the method, but it throws only garbage ...  I tried.
Also ROS plugins does not use it and compute the pointCloud manually. You can see here:
http://docs.ros.org/jade/api/gazebo_plugins/html/gazebo__ros__openni__kinect_8cpp_source.html#l00184